.github/actions/install-poetry/action.yml

```diff
@@ -35,3 +35,4 @@ runs:
         virtualenvs-create: true
         virtualenvs-in-project: true
         installer-parallel: true
+        virtualenvs-path: ~/.venv-${{ runner.os }}-${{ matrix.python-version }}
```

Comment:
I think we could keep using the normal .venv here, this way we will have less problems down the line (as otherwise people need to be aware which path the .venv has)

Reply:
I was running some experiments using the same `venv` was bringing some issues. The caching was also kind of a problem here.

Reply:
I'm not sure that this setting is useful, considering that we also use the `virtualenvs-in-project: true`. I couldn't find the exact behaviour when you define both `virtualenvs-in-project: true` and `virtualenvs-path`, but reading through the lines of the docs, it seems like the `virtualenvs-in-project: true` takes priority. But anyway, we shouldn't use both IMO. I would stick only with `virtualenvs-in-project: true`, since in some build & test pipelines we rely on the venv to be in the project.